react/src/pages/NotificationPopover.tsx

```diff
@@ -66,7 +68,9 @@ export default function NotificationPopover({ lastLoginTime }: NotificationPopov
         <div>
             <IconButton onClick={handlePopoverOpen}>
                 <Tooltip title="See notifications" arrow>
-                    <NotificationsActive fontSize='large' style={{fill: "white"}} />
+                    <Badge badgeContent={numNotification} color="secondary">
```

Comment:
You don't need another state variable to track this. You can just use `analyses.length` here.